docs/variants/qemu_q35/releases.md

```diff
@@ -3,15 +3,39 @@
-- Option to reset Secure Boot keys to defaults.
-- Dasharo System Features.
-- Dasharo setup password.
-- OPAL disk password support.
-- SATA disk password support.
-- SMM Feature enabled.
+- Configurable boot order
+- Configurable boot options
+- [Custom boot menu keys](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/303-custom-boot-menu-key/)
+- [UEFI shell](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/30P-uefi-shell/)
+- [UEFI Secure Boot](https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/)
+- [TPM Support](https://docs.dasharo.com/unified-test-documentation/dasharo-security/200-tpm-support/)
+- [PS/2 Controller enable/disable option](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#chipset-configuration)
+- [Watchdog configuration menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#chipset-configuration)
+- [Early boot DMA protection menu option](https://docs.dasharo.com/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection/)
+- [Intel ME disable support and menu options](https://docs.dasharo.com/unified-test-documentation/dasharo-security/20F-me-neuter/)
+- [Dasharo setup password](https://docs.dasharo.com/unified-test-documentation/dasharo-security/20R-uefi-setup-password/)
+- [SED/OPAL disk password support](https://docs.dasharo.com/unified-test-documentation/dasharo-security/208-opal-disk-password-support/)
+- [SATA disk password support](https://docs.dasharo.com/dasharo-menu-docs/device-manager/#hdd-security-configuration)
+- SMM BIOS Write Protection support and enable/disable option
+- [USB stack and mass storage enable/disable option](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#usb-configuration)
+- [Firmware Update Mode feature](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+- [One of the two fan profiles can now be selected in Setup Menu](https://docs.dasharo.com/unified/novacustom/fan-profiles/)
+- [Setup menu option for switching between S0ix and S3 suspend mode](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#power-management-options)
+- Serial Port Configuration menu
+- [Wi-Fi / Bluetooth module disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+- [Built-in webcam disable option in setup menu](https://docs.dasharo.com/unified/novacustom/camera-switch/)
+- [Battery threshold options in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#power-management-options)
+- [PCIe Configuration menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#pcipcie-configuration)
+- [Memory configuration menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#memory-configuration)
+
+### Binaries
+
+https://github.com/Dasharo/edk2/releases
```

Comment:
The idea would be to publish releases simply via github release for lowest effort. Let me know what you think of that.

Reply:
@macpijan I'm good with that, although I want to ensure we have the same quality criteria as regular Dasharo releases (public test results, hashes, signatures, release notes), but on GH.

Reply:
For signatures we would need to add signing key to github, which might not be the best idea.
Or maybe there is a way to add signatures to release later.

Reply:
@macpijan it depends on what signature you talking about - we had this discussion with @miczyg1 elsewhere. What I have in mind is a hash signing what is standard for us now in all releases and happen out of any CI.

Reply:
Yes, hash signing. If this should be part of CI, key must be stored somewhwere as well.

Reply:
@macpijan did I suggest that signing should be part of CI? None of our projects use automatic signing in CI, so why we would do it here? The signing could be a manual and the last step in the whole release process. When signing, the key owner would look at test results to confirm that the given hash contains given properties. Part of the CI can be binary publishing, signing should happen separately.

Reply:
Let's try that.